Backend/dev/migrations-read-only/rider-app/partner_organization.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE atlas_app.partner_organization ();
+
+ALTER TABLE atlas_app.partner_organization ADD COLUMN api_key_hash text NOT NULL;
```

Comment:
Let's also keep an encrypted version of the key, so that we can retrieve it later if required